### PR TITLE
tnozicka: add rpms

### PR DESCRIPTION
**Pull request by sallyom**
@deads2k

### CODE REVIEW

File: oc.spec [hunk 1]
```diff
@@ -19,7 +19,12 @@ Summary:        OpenShift client binaries
 License:        ASL 2.0
 URL:            https://%{import_path}
 
+# If go_arches not defined fall through to implicit golang archs
+%if 0%{?go_arches:1}
 ExclusiveArch:  %{go_arches}
+%else
+ExclusiveArch:  x86_64 aarch64 ppc64le s390x
```
Comment (sallyom) [inline oc.spec]: took this from origin's spec, can clean up later if this isn't exactly what we need; this fixes the CI failure though.

Comment (tnozicka) [reply oc.spec]: hm, I have tried manually putting `x86_64` there yesterday, but it failed with the same error

**Comment by tnozicka** [comment]
hm, the Makefile doesn't include the rpm.mk at all

**Comment by sallyom** [comment]
closing, this merged, yay! https://github.com/openshift/oc/pull/36

**Pull request closed without merging**